news.xml

```diff
@@ -674,7 +674,7 @@ $ nix-store -qR /run/current-system | grep openssl
     </title>
     <description>
       A sprint focused on NixOS and <a href="http://kotti.pylonsproject.org/">Kotti</a> will be held <a
-        href="http://www.coactivate.org/projects/zidanca-sprint-2013/project-home">22-26
```

Comment:
the link still works

Reply:
static site that won’t change in the future.